.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: node_js
 node_js:
   - "1.8"
-  - "2.5"
-  - "3.3"
-  - "4.4"
+  - "4"
```

Comment:
As said here https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Available-Versions
This will select the last "4.x", "6.x" and latest version